test/gtest/ucs/test_async.cc

```diff
@@ -599,10 +599,6 @@ UCS_TEST_P(test_async, warn_block) {
     for (int i = 0; i < warn_count; ++i) {
         UCS_TEST_MESSAGE << "< " << m_warnings[i] << " >";
     }
-
-    if (GetParam() != UCS_ASYNC_MODE_POLL) {
```

Comment:
why?

Reply:
I remove above line for fixing this error.
I removed `ucs_warn` from `ucs_spinlock_destroy()` So, error counter returns invalid count. I'll change expect value to zero.

```
14:13:58 [ RUN      ] thread_spinlock/test_async.warn_block/0
14:13:58 /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-test-althca/worker/0/contrib/../test/gtest/ucs/test_async.cc:604: Failure
14:13:58 Expected: (warn_count) >= (1), actual: 0 vs 1
14:13:58 
[  FAILED  ] thread_spinlock/test_async.warn_block/0, where GetParam() = 1 (0 ms)
```

Ref: https://github.com/openucx/ucx/pull/3939#issuecomment-516255178